extension/src/server/GradleServer.ts

```diff
@@ -70,7 +70,10 @@ export class GradleServer {
             sendInfo("", {
                 kind: "GradleServerEnvMissing",
             });
-            await vscode.window.showErrorMessage(NO_JAVA_EXECUTABLE);
+            const selection = await vscode.window.showErrorMessage(NO_JAVA_EXECUTABLE, INSTALL_JDK);
```

Comment:
I just realized that before add the install jdk option, we need to check if java pack extension is installed